### PR TITLE
chore: upgrade to latest storybook PUL-1366

### PR DESCRIPTION
**Pull request by mshaaban0**
# Purpose of PR

* Upgrade to latest storybook
* Remove unnecessary add-ons
* Remove unused components inside storybook

## Todo
- [x] fix: preview text styles regression

### CODE REVIEW

**Comment by gui-santos** [comment]
@mshaaban0 is this still WIP?

It would be cool to get storybook updated 🙌

**Comment by mshaaban0** [comment]
@gui-santos I'll pick this up now 👍

File: scripts/.storybook/main.js [hunk 1]
```diff
@@ -7,8 +7,6 @@ module.exports = {
     '@storybook/addon-essentials',
     '@storybook/addon-knobs',
     '@storybook/addon-a11y',
-    'storybook-prop-types-addon',
-    'storybook-addon-jsx',
```
Comment (denkristoffer) [inline scripts/.storybook/main.js:11]: Can we remove these from `package.json` or do we still need them?

Comment (mshaaban0) [reply scripts/.storybook/main.js:11]: Good catch, I thought I removed them, rebase, sorry!

File: scripts/.storybook/manager-head.html [hunk 2]
```diff
@@ -1 +1 @@
-<link rel="icon" type="image/png" href="/logo.png" sizes="32x32" />
+<link rel="icon" type="image/png" href="/favicon.png" sizes="32x32" />
```
Comment (denkristoffer) [inline scripts/.storybook/manager-head.html:1]: I don't see this file anywhere, is it a Storybook default?

**Review by denkristoffer — APPROVED** [review]

**Review by gui-santos — APPROVED** [review]
YES!

File: scripts/.storybook/manager-head.html (hunk 2 above)
Comment (mshaaban0) [reply scripts/.storybook/manager-head.html:1]: not a default, I renamed `logo.png` in public to `favicon.png` but slipped away during rebase, thanks 😅

**Pull request merged**